github/GithubException.py

```diff
@@ -38,7 +38,7 @@ class GithubException(Exception):
     Some other types of exceptions might be raised by underlying libraries, for example for network-related issues.
     """
 
-    def __init__(self, status, data, headers):
+    def __init__(self, status, data, headers=None):
```

Comment:
I had a 404 while testing and had an exception because the headers parameters was not passed.

Reply:
I think fixing the calling code as in #2079 is better than changing the signature.